content/W2017/W17-event-beckwith-talk.md

```diff
@@ -0,0 +1,29 @@
+Title: Tech Talk: Hotspot Virtual Machine's Execution Engine by Monica Beckwith
+Date: 2017-03-14
+Category: Events
+Tags: events, academic
+Slug: beckwith-tech-talk-w17
+Author: Monica Xu
+Summary: Come listen about the amazing work Monica Beckwith is doing!
```

Comment:
The summary is basically like a headline, so we want to include more information about the talk. Have a look at the Leigh Honeywell post, there's a good example there.